letsencrypt/client/ddns/ddns.py

```diff
@@ -0,0 +1,85 @@
+"""generic DDNS authenticator (RFC 2136)"""
+
+import logging
+import subprocess
+
+import zope.interface
+
+from letsencrypt.client import challenge_util
+from letsencrypt.client import interfaces
+
+
+class DDNS(object):
+    """DDNS authenticator.
+
```

Comment:
:) I didn't open the whole file.  Yes the DDNS docstring should just be one line. 

(removing the first line whitespace does keep it consistent with the rest of the codebase though too)  Thanks @ThomasWaldmann!